docsk8s/administering/ls-k8s-stack-monitoring.asciidoc

```diff
@@ -1,14 +1,122 @@
 [[ls-k8s-stack-monitoring]]
-=== Set up stack monitoring
+=== Stack Monitoring with ECK
 
 WARNING: This documentation is still in development and may be changed or removed in a future release.
 
-In order to prepare your environment to be production ready, you'll need to set up secure communication between each of your Elastic resources.
+An important step to making your environment production ready is to configure stack monitoring. Monitoring metrics can be sent to an external resource, such as {ess} or {eck}, so that in the event that any components of your environment become unresponsive, your monitoring data is available.
 
-An important step to making your environment production ready is to configure stack monitoring. Monitoring logs and metrics data can be sent to an external resource, such as {ess} or {eck}, so that in the event that any components of your environment become unresponsive, your monitoring data is available.
+To enable {logstash-ref}/monitoring-with-metricbeat.html[Stack Monitoring] for {ls}, you need {metricbeat} to collect {ls} metrics, {es} to store the metrics and {kib} to view the result.
 
-Follow these steps to configure monitoring:
+NOTE: The prerequisite of the example is having ECK installed. Checkout <<qs-set-up>>
 
-. Step 1
-. Step 2
-. Step 3
\ No newline at end of file
+For these examples, we will be modifying the Beats stack monitoring link:https://github.com/elastic/cloud-on-k8s/blob/main/config/recipes/beats/stack_monitoring.yaml[recipe] from the ECK examples, which initiates a production {es} cluster, a monitoring {es} cluster, {filebeat}, {metricbeat}, a production Kibana and a monitoring Kibana. It monitors {es} and Kibana and send metrics to monitoring cluster.
```

Comment:
Maybe we should have our own recipe in our own repo, rather than modifying a pre-existing recipe?

This can be a follow-up task if necessary

Reply:
The recipe from Beats is pretty nice and they keep it up-to-date. It can be a follow-up for improvement.